src/palace/manager/api/admin/form_data.py

```diff
@@ -39,23 +39,26 @@ def get_settings_dict(
         a dictionary that we can use to update the settings.
         """
         return_data: dict[str, Any] = {}
-        for field in settings_class.__fields__.values():
-            if not isinstance(field.field_info, FormFieldInfo):
-                continue
-            form_item = field.field_info.form
+        for field_name, field_info in settings_class.model_fields.items():
+            assert isinstance(
+                field_info, FormFieldInfo
+            ), f"Expected FormFieldInfo, got {field_info.__class__}"
```

Comment:
This used to just silently not include non `FormFieldInfo` settings. I switched this to be an assertion so we fail in this case. It shouldn't ever be the case, and if it is, I think we probably want to fail so we can understand what happened.